modules/search-service-impl/src/main/java/org/opencastproject/search/endpoint/SearchRestService.java

```diff
@@ -261,6 +270,36 @@ public Response getEpisode(@QueryParam("id") String id, @QueryParam("q") String
       }
     }
 
+    if (seriesName != null && seriesId != null) {
+      return Response.status(Response.Status.BAD_REQUEST).entity("invalid request, both 'sid' and 'sname' specified")
+              .build();
+    }
+
+    if (seriesName != null) {
+      DublinCoreCatalogList result;
+      try {
+        result = seriesService.getSeries(new SeriesQuery().setSeriesTitle(seriesName));
+      } catch (SeriesException e) {
```

Comment:
In the past, we found it dangerous to tie the Search Service REST endpoint, that usually deploys to engage, to a service that usually lives on the Admin node. It caused the Engage player to tightly dependent on the Admin node being up and accessible from the Engage node.

Reply:
This looks to me like an out of date comment, but just in case: this is bound to the impl/remote, not the rest endpoint.